indy_node/server/req_handlers/query_req_handlers/get_attribute_handler.py

```diff
@@ -0,0 +1,5 @@
+from indy_node.server.query_req_handler import QueryReqHandler
+
+
+class GetAttributeHandler(QueryReqHandler):
```

Comment:
It should extend ReadRequestHandler from Plenum, shouldn't it?

Reply:
yes